src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -35,6 +35,11 @@ public PopupPanel(Popup popup)
 
 		protected override Size MeasureOverride(Size availableSize)
 		{
+			if (Visibility == Visibility.Collapsed)
+			{
+				availableSize = new Size(); // 0,0
```

Comment:
really strange that this is necessary

Reply:
I see - can you add that as a comment

Reply:
Right !

Reply:
Also - why have we not needed it up until _now_? Something special about the `Picker`?